### PR TITLE
Adds cogscarabs to clock cult

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Servants of ratvar can now create cogscarab shells with a Script scripture. Adding a soul vessel to one will produce a small, drone-like being with an inbuilt proselytizer and tools.
imageadd: Clockwork mobs have their own speechbubble icons.
/:cl:

Cogscarabs can communicate over the Hierophant Network with :b and are effectively drones, have hands, cannot ventcrawl, and are dense.
Rips out the drone conversion code in the interest of having less problems to deal with.
Merge token; #18673
space station 13 2016-06-17 161455

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@ This file's folder contains:
 		return 0
 	if(M.mind.enslaved_to)
 		return 0
-	if(isdrone(M))
+	if(isdrone(M) && !istype(M, /mob/living/simple_animal/drone/cogscarab))
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: :/

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: Very bleh, yes. I'm not sure if there's really an alternative here?

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 2]
```diff
@@ -87,3 +87,51 @@
...
+
+/mob/living/simple_animal/drone/cogscarab/New()
+	. = ..()
+	qdel(access_card) //we don't have free access
+	access_card = null
+	verbs -= /mob/living/simple_animal/drone/verb/check_laws
+	verbs -= /mob/living/simple_animal/drone/verb/toggle_light
+	verbs -= /mob/living/simple_animal/drone/verb/drone_ping
+	verbs -= /mob/living/simple_animal/drone/verb/toggle_statics
+	for(var/datum/action/generic/drone/select_filter/SF in actions) //clear all the drone shit we don't need
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: how about you make ```seeStatic = 0``` not add the action in the first place.

File: code/game/gamemodes/clock_cult/clock_cult.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: just roll them into isdrone

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: what
goof no, they ARE drones and should return TRUE if isdrone is checking them, but I don't want them to be ineligible for being servants.

**Comment by KorPhaeron** [comment]
Do they have full access of the station like drones do?

**Comment by ChangelingRain** [comment]
Nope.
They've got no access except for what being a servant of ratvar gets you.

**Comment by KorPhaeron** [comment]
Neat

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 3]
```diff
@@ -696,6 +696,22 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 
+/datum/clockwork_scripture/create_object/anima_fragment //Anima Fragment: Creates an empty anima fragment
+	descname = "Constructor Soul Vessel Shell"
```
Comment (Core0verload) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Seems like you forgot to change the path.

**Comment by Jordie0608** [comment]
Cool, but I'm not too sure about the balance of giving the clock cult a walking x-ray scout.

**Comment by ChangelingRain** [comment]
If you're concerned about the turf vision/object vision it's easy enough to change it to turf-only or just remove it.

**Comment by Jordie0608** [comment]
I can't really say for sure without seeing it in action. Do they have the same mob vision as normal drones?

**Comment by ChangelingRain** [comment]
They can see and mess with mobs; I wouldn't say they'd last long doing so, though, because they've got 60 health and are hit by projectiles.

**Pull request merged**